src/pudl/extract/ferc1.py

```diff
@@ -51,29 +51,19 @@
 import csv
 import importlib
 import io
-import json
-import time
-from datetime import datetime
 from pathlib import Path
 
 import dbfread
 import pandas as pd
 import sqlalchemy as sa
 from dbfread import DBF
-from ferc_xbrl_extractor import xbrl
-from ferc_xbrl_extractor.instance import InstanceBuilder
 from sqlalchemy import or_
 
 import pudl
 from pudl.helpers import get_logger
 from pudl.metadata.classes import DataSource
 from pudl.metadata.constants import DBF_TABLES_FILENAMES
-from pudl.settings import (
-    Ferc1DbfSettings,
-    Ferc1DbfToSqliteSettings,
-    Ferc1XbrlSettings,
-    Ferc1XbrlToSqliteSettings,
-)
+from pudl.settings import Ferc1DbfSettings, Ferc1DbfToSqliteSettings, Ferc1XbrlSettings
```

Comment:
I thought we had unified the `Ferc1DbfSettings` and `Ferc1XbrlSettings` into a single `Ferc1Settings` that just specifies the desired PUDL tables and years of data, and which can look up the required input tables.  Or maybe that's only on the `xbrl_steam` branch?

Reply:
I think that change is only on the `xbrl_steam` branch. It will definitely be nice to get some of these branches merged into `xbrl_integration`, so we don't have to track which changes are on which branch

Reply:
Ah okay sorry. Yes, let's get everything in there!